compiler/parser/support.go

```diff
@@ -33,6 +33,20 @@ func makeArgMap(args interface{}) (interface{}, error) {
 	return m, nil
 }
 
+func makeTemplateExprChain(in interface{}) interface{} {
```

Comment:
Maybe someday we can get rid of this and just use makeBinaryExprChain since that's so similar.